envoy/datadog_checks/envoy/envoy.py

```diff
@@ -59,9 +53,11 @@ def check(self, instance):
             self.caching_metrics = instance.get('cache_metrics', True)
 
         try:
-            response = requests.get(stats_url, auth=auth, verify=verify_ssl, proxies=proxies, timeout=timeout)
+            response = self.http.get(stats_url)
         except requests.exceptions.Timeout:
-            msg = 'Envoy endpoint `{}` timed out after {} seconds'.format(stats_url, timeout)
+            msg = 'Envoy endpoint `{}` timed out after {} seconds'.format(
+                stats_url, timeout=int(instance.get('timeout', 20))
```

Comment:
Should be `STANDARD_FIELDS['timeout']` instead ? from https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/utils/http.py#L54

```python
                stats_url, timeout=instance.get('timeout', STANDARD_FIELDS['timeout'])
```

Reply:
Rather `self.http.options['timeout']`